packages/app/cypress/e2e/debug.cy.ts

```diff
@@ -77,7 +77,7 @@ describe('App - Debug Page', () => {
       cy.findByTestId('debug-header-branch').contains('main')
       cy.findByTestId('debug-header-commitHash').contains('e9d176f')
       cy.findByTestId('debug-header-author').contains('Lachlan Miller')
-      cy.findByTestId('debug-header-createdAt').contains('01:18')
+      cy.findByTestId('debug-header-createdAt').contains('01m 18s')
```

Comment:
I notice some logic specific for hours in the DebugHeaderPage - any test around that logic? This one only seems to use minutes/seconds. You could just modify the JSON fixtures.